docs/language/learn-ql/javascript/type-tracking.rst

```diff
@@ -1,5 +1,5 @@
-Tutorial: API modelling using type tracking
-===========================================
+Using type tracking for API modeling
```

Comment:
Nitpicking: If the docs don't use American English, this should probably be `modelling` with two `l`s?

Reply:
Thanks for highlighting this. The docs aim to use American English, so this is correct.